build.sbt

```diff
@@ -22,7 +22,7 @@ crossScalaVersions := Seq("2.12.8", "2.11.12")
 
 scalaVersion := crossScalaVersions.value.head
 
-resolvers += "Apache Spark snapshot repository" at "https://repository.apache.org/content/repositories/snapshots"
+resolvers += Resolver.mavenLocal
```

Comment:
you should probably revert this change. The hope is that all fixes will be in Spark 3.0 some time soon